### DISCUSSION TITLE
Update only after answering question to do so?

### DISCUSSION THREAD

**Discussion by timoreichen** [shannah/jdeploy#125] [Q&A]
Now there is "Auto update settings" .. On/Bugs..Never. Is there a way to do it after have asked the user? Like "Hey, there is a new version out. Would like to update now?" If not, the questions arise always on start but the user can choose when to update.

**Comment by shannah** [comment]
Currently this isn't an option.  Wouldn't be hard to add this if it is needed.  You can add a feature request to the issue tracker.
